packages/api/test/e2e/queries.e2e.ts

```diff
@@ -132,4 +132,21 @@ describe('e2e queries', () => {
        done();
     });
   });
+
+  describe('Staking account derived query', () => {
+    it("Get staking account details", async done => {
+      const stashId = '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY'; // alice_stash
+      const stakingAccount = await api.derive.staking.query(stashId);
+      expect(stakingAccount.accountId.toString()).toBe(stashId);
+      expect(stakingAccount.controllerId.toString()).toBe(alice.address);
+      expect(stakingAccount.nominators).toHaveLength(0); // Initially no nominators
+      expect(stakingAccount.rewardDestination.isStaked).toBeTruthy();
+      expect(stakingAccount.stakers).toBeDefined();
+      expect(stakingAccount.stakingLedger.stash.toString()).toBe(stashId);
+      expect(stakingAccount.validatorPrefs.toString()).toBe("commission,0");
```

Comment:
with type hinting (`ValidatorPrefs`) you should be able to check it like: `(stakingAccount.validatorPrefs.commission.toNumber()).toBe(0);`